### DISCUSSION TITLE
PHP Executable not found

### DISCUSSION THREAD

**Discussion by jdevries14** [pimcore/pimcore#10198] [Q&A / Support]
Hi, when I try to create an image thumbnail I get the following.

<img width="715" alt="Screenshot 2021-08-30 at 17 14 34" src="https://user-images.githubusercontent.com/44669189/131362151-3080f1b3-f3cc-4600-bc89-cf6a6726294b.png">

But when I run `bin/console pimcore:system:requirements:check` it gives me a check on PHP

<img width="492" alt="Screenshot 2021-08-30 at 17 15 13" src="https://user-images.githubusercontent.com/44669189/131362256-93d74d53-e1d5-414f-8286-4e61d1049fea.png">
<img width="482" alt="Screenshot 2021-08-30 at 17 15 30" src="https://user-images.githubusercontent.com/44669189/131362294-18ae229a-126c-479e-8f2c-c251d11bd85a.png">

So what is going wrong? Also when running `php -v` it gives me output back in the terminal

**Comment by dvesh3** [comment]
@jdevries14 have you tried setting executable path like mentioned here https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Additional_Tools_Installation.html#page_Additional-Tools-Installation?

**Comment by ramundomario** [comment]
Are you using PHP-FPM, right? 
if so, by default, PHP-FPM clears the environment variables (including the PATH variable).
There are two possible configurations in pool (on ubuntu: `/etc/php/8.0/fpm/pool.d/www.conf`)

- Export  only `PATH` variable:
`env[PATH] = $PATH`
- Disable `clear_env` configuration:
```
; Clear environment in FPM workers
; Prevents arbitrary environment variables from reaching FPM worker processes
; by clearing the environment in workers before env vars specified in this
; pool configuration are added.
; Setting to "no" will make all environment variables available to PHP code
; via getenv(), $_ENV and $_SERVER.
; Default Value: yes
clear_env = no
```
